app/views/trains/index.html.slim

```diff
@@ -1,5 +1,7 @@
 h1 Trains Index
 = link_to 'New train', new_train_path
+br
+= notice
```

Comment:
left notes on the delete PR that are applicable here too, if we can style this with appropriate styles it would be good.